pandas/io/gbq.py

```diff
@@ -108,6 +112,17 @@ def read_gbq(query, project_id=None, index_col=None, col_order=None,
     pandas.DataFrame.to_gbq : Write a DataFrame to Google BigQuery.
     """
     pandas_gbq = _try_import()
+
+    if dialect is None:
+        dialect = "legacy"
+        warnings.warn(
+            'The default value for dialect is changing to "standard" in a '
+            'future version of pandas-gbq. Pass in dialect="legacy" to '
+            "disable this warning.",
+            FutureWarning,
+            stacklevel=2,
+        )
+
```

Comment:
Do we have a good way to test this by any chance?

Reply:
I've got a test in the pandas-gbq repo:

https://github.com/pydata/pandas-gbq/blob/998b6cfe953613bf5ada6253a7d3550837a2d169/tests/unit/test_gbq.py#L367-L369

I suppose I should include a similar test here?

Reply:
If it's possible to include in some shape or form, I would definitely be 👍 for it.

Reply:
Added in https://github.com/pandas-dev/pandas/pull/22557/commits/07c2eb80875f73fbed8a38b84cd0d5f34246868c